librz/core/cmd_descs/cmd_system.yaml

```diff
@@ -0,0 +1,71 @@
+# SPDX-FileCopyrightText: 2021 RizinOrg <info@rizin.re>
+# SPDX-License-Identifier: LGPL-3.0-only
+---
+name: cmd_system3
+commands:
+  - name: "!"
+    cname: system3
+    summary: Runs given commands in system(3)
+    args:
+      - name: command
+        type: RZ_CMD_ARG_TYPE_STRING
+        optional: true
+        no_space: true
+      - name: args
+        type: RZ_CMD_ARG_TYPE_STRING
+        flags: RZ_CMD_ARG_FLAG_ARRAY
+        optional: true
+    details:
+      - name: Examples
+        entries:
+          - text: "!"
+            arg_str: "ls"
+            comment: "executes the 'ls' command via system(3)"
+          - text: "!"
+            arg_str: "echo $RZ_SIZE"
+            comment: "executes the 'echo' command via system(3) and shows the display file size"
+  - name: "!!"
+    cname: system3_to_cons
+    summary: Runs a given commands in system(3) and pipes stdout to rizin
+    args:
+      - name: command
+        type: RZ_CMD_ARG_TYPE_STRING
+        no_space: true
+      - name: args
+        type: RZ_CMD_ARG_TYPE_STRING
+        flags: RZ_CMD_ARG_FLAG_ARRAY
+        optional: true
+    details:
+      - name: Examples
+        entries:
+          - text: "!!"
+            arg_str: "ls~txt"
+            comment: "executes the 'ls' command via system(3) and grep for 'txt'"
+  # These commands should me moved from ! to somewhere else.
+  - name: "!="
```

Comment:
where did you get these from?

Reply:
It was a bit confusing having `!<cmd>` and `!<num>` under the same command,
therefore i moved the `show list history` and `execute history N` under `!=`

Btw, I added all my reasoning in my commit message; you can click on the 3 dots and read the body message
![image](https://user-images.githubusercontent.com/561184/122897367-3aa71080-d34a-11eb-9bcc-533bc377a7a1.png)

Reply:
:+1: